src/cirrus/configuration.py

```diff
@@ -337,3 +337,15 @@ def get_chef_auth():
         'chef_client_user': chef['chef-client-user'],
         'chef_client_keyfile': chef['chef-client-keyfile']
     }
+
+
+def get_github_api_base():
+    """
+    Return the github.api_base URL, or https://api.github.ibm.com if not
+    configured.
+    """
+    try:
+        url = load_configuration()['github']['api_base'].rstrip('/')
+    except KeyError:
+        url = 'https://api.github.ibm.com'
```

Comment:
print out some warning message like, "Failed to load api_base from github config, using default: https://api.github.ibm.com"